js/whitebit.js

```diff
@@ -615,11 +694,424 @@ module.exports = class whitebit extends Exchange {
         return this.status;
     }
 
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        let method = undefined;
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+            'side': side,
+            'amount': this.numberToString (this.amountToPrecision (symbol, amount)),
+        };
+        if (type === 'market') {
+            // for this order type, the amount in the stock currency to buy/sell
```

Comment:
Is this an expectable behavior from an exchange? When the side is "buy" the amount is in base currency, otherwise, it's in the quote currency.

Reply:
> Is this an expectable behavior from an exchange? When the side is "buy" the amount is in base currency, otherwise, it's in the quote currency.

Yes:

- search `createMarketBuyOrderRequiresPrice` in the existing exchanges for reference
- https://github.com/ccxt/ccxt/wiki/Manual#market-buys

Reply:
done